Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -249,37 +254,40 @@ extension TokenSyntax: ExpressibleByTokenSyntax {
 
 // MARK: - Syntax buildable expressible by conformances
 
-% for protocol, conformances in SYNTAX_BUILDABLE_EXPRESSIBLE_BY_CONFORMANCES.items():
-%   if 'ExpressibleByConditionElementList' in conformances:
-extension ${protocol} {
-  public func createConditionElementList() -> ConditionElementList {
-    ConditionElementList([self])
+% for protocol, conformances in SYNTAX_COLLECTION_EXPRESSIBLE_BY_CONFORMANCES.items():
+%   for conformance in conformances:
+extension ExpressibleBy${protocol} {
+  public func create${conformance}() -> ${conformance} {
+    ${conformance}([self])
   }
 }
 
 %   end
-%   if 'ExpressibleByConditionElement' in conformances:
-extension ${protocol} {
+% end
+
+% for protocol, conformances in SYNTAX_BUILDABLE_EXPRESSIBLE_BY_CONFORMANCES.items():
+%   if 'ConditionElement' in conformances:
+extension ExpressibleBy${protocol} {
```

Comment:
Not part of this PR, but couldn’t we also generate these `extension` implementations now that syntax collection conformances live somewhere else?

Reply:
I would be nice! 
But the thing is, that the init method different for each type. 

We can enrich the `protocols.py`file with the needed information.

Reply:
Why is that? Am I missing something obvious? Shouldn’t the following work?

```swift
extension ExpressibleBy${protocol} {
   public func create${expressible_as_protocol}() -> ${expressible_as_protocol} {
     ${expressible_as_protocol}(${retrieve only non-optional child name here}: self)
   }
 }

```

Reply:
```swift
extension ExpressibleByDeclBuildable {
  public func createCodeBlockItem() -> CodeBlockItem {
    CodeBlockItem(item: self)
  }
}

extension ExpressibleByDeclBuildable {
  public func createMemberDeclListItem() -> MemberDeclListItem {
    MemberDeclListItem(decl: self)
  }
}
```

Yes. Sorry bad explanation 😅
The `${retrieve only non-optional child name here}` do we need to find somehow. 
Not sure how to do, but I will look into it, and then we can take it from there. 

It is possible, somehow 😬